catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/SeedCommandTest.java

```diff
@@ -205,6 +212,19 @@ public void testDoesNotDownloadCachedResource() throws Exception {
         verify(catalogFramework, times(2)).query(any(QueryRequest.class));
     }
 
+    @Test
+    public void testDoesNotSkipResources() throws Exception {
+
+        int maxPageSize = NumberUtils.toInt(System.getProperty("catalog.maxPageSize"), 1000);
+        int resourceLimit = maxPageSize * 2;
+
+        mockMultipleQueries(maxPageSize, resourceLimit);
+
+        seedCommand.executeWithSubject();
+        assertThat(consoleOutput.getOutput(),
+                containsString(resourceLimit + " resource download(s) started."));
```

Comment:
✏️ Often it helps to place an `expected` string at the start of the unit test and avoid constructing a string inside the assert statement (construct to its own local variable in the line above it might be a bit clearer).